stdlib/runtime/bigint.gr

```diff
@@ -1827,10 +1827,10 @@ let baseCaseDivRem = (a: WasmI32, b: WasmI32, result: WasmI32) => {
   let n = getHalfSize(b)
   let m = getHalfSize(a) - n
   let qsize = (if (WasmI32.eqz(WasmI32.and(m + 1n, 1n))) {
-        m + 1n
-      } else {
-        m + 2n
-      }) >>
+    m + 1n
+  } else {
+    m + 2n
+  }) >>
```

Comment:
I think you mistook my comment below this to indicate that I didn't want this `if` statement indented, but it was actually about putting the `>>` and `1n` on the same line since they don't actually wrap.

I don't know how I feel about you taking away the indent. @ospencer thoughts?

Reply:
To me it seems like it should be indented the way it was before?

Reply:
@marcusroberts this was the line that needed to be reverted